tests/test_models/test_simple_models.py

```diff
@@ -55,6 +55,26 @@ def df():
     return tsds
```

Comment:
May you also add test checking the correctness of decomposition like in for SMA [here](https://github.com/tinkoff-ai/etna/blob/83db9b70cc5e7e9e7ad9e2b6cf2d1811a404900a/tests/test_models/test_simple_models.py#L787)